data/sql/updates/pending_db_world/rev_1677249657204460100.sql

```diff
@@ -0,0 +1,7 @@
+-- Thule Ravenclaw - remove ranged movement
+DELETE FROM `smart_scripts` WHERE `entryorguid` = 1947;
```

Comment:
```suggestion
DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = 1947;
```

Otherwise it could delete other SAI of other types. I know, it's unlikely but it's better safe than sorry.
And sorry I didn't see it before